pkg/workflows/wasm/host/adapters/beholder/emitter.go

```diff
@@ -0,0 +1,17 @@
+package beholder
+
+import (
+	"context"
+
+	b "github.com/smartcontractkit/chainlink-common/pkg/beholder"
+)
+
+type emitter struct{}
+
+func NewEmitter() *emitter {
+	return &emitter{}
+}
+
+func (e *emitter) Emit(msg string, labels map[string]any) error {
```

Comment:
Nit/style: this is small enough that it could live in a separate file in host, but that may be a preference more than anything (I like to keep things clustered together as much as possible)